build.gradle

```diff
@@ -254,7 +254,7 @@ project(':core') {
 
   dependencies {
     compile project(':clients')
-    compile "$slf4jlog4j"
+    compile project(':log4j-appender')
```

Comment:
Ok, if we do this, we should remove the line where this jar is manually copied to libs.